server/sonar-server/src/main/java/org/sonar/server/user/ws/GroupsAction.java

```diff
@@ -134,11 +128,12 @@ private void writePaging(JsonWriter json, Paging paging) {
       .prop("total", paging.total());
   }
 
-  private String getMembership(@Nullable String selected) {
+  private String getMembership(String selected) {
+    SelectionMode selectionMode = SelectionMode.fromParam(selected);
     String membership = GroupMembershipQuery.ANY;
-    if (SELECTION_SELECTED.equals(selected)) {
+    if (SELECTED == selectionMode) {
```

Comment:
It's the enum value which is used here; maybe I should remove the static import to make it more obvious.